### ISSUE TITLE
NuGet scripts should be compatible with PowerShell 2.0

### ISSUE THREAD

**Issue by cuoilennaocacban** [SonarSource/sonar-csharp#244]
when I install the package in a project using Visual Studio 2015, this error show up

```
You must provide a value expression on the right-hand side of the '-' operator.
At [ProjectFolder]\packages\SonarAnalyzer.CSharp.5.9.0.1001\tools\install.ps1:8 char:27
+ if ($project.DTE.Version - <<<< In '14.0', '15.0')
    + CategoryInfo          : ParserError: (:) [], ParseException
    + FullyQualifiedErrorId : ExpectedValueExpression
```

The problems is inside the install.ps1 script.

**Comment by Evangelink** [comment]
Hi @cuoilennaocacban 

Could you tell which version of powershell is installed on your machine (where the failure happens)? You can run the following command from within the powershell invite: `$PSVersionTable.PSVersion`.

**Comment by cuoilennaocacban** [comment]
2.0
Major: 2
Minor: 2
Build: -1
Revision: -1

And I'm using Windows 7 build 7601 service pack 1

**Comment by Evangelink** [comment]
The current `install.ps1` and `uninstall.ps1` are using `-In` and `-NotIn` which are commands available from Powershell 3.0. We should rewrite the condition to use only syntax from Powershell 2.0
